django_check_seo/checks_list/check_description.py

```diff
@@ -207,7 +207,7 @@ def run(site):
         site.warnings.append(too_much_meta)
     # perfect
     elif number_meta_description == 1:
-        site.problems.append(no_meta_description)
+        # site.problems.append(no_meta_description)
```

Comment:
Why is this check commented?

Reply:
*edit (I re-read my comment and found it confusing):* you can remove the line instead of just commenting it :)

Reply:
Yes, I actually just tested removing that line, didn't realize it'd end up in the PR, as it's not really related to ugettext. 
Regardless, I think it's a bug and I removed it in 9d8945d9bd3a5f2cb0639be154911874abc5f56b